src/main/java/com/khomishchak/cryptoportfolio/model/enums/ExchangerCode.java

```diff
@@ -1,7 +1,9 @@
 package com.khomishchak.cryptoportfolio.model.enums;
 
 public enum ExchangerCode {
+    // To be implemented:
     BINANCE,
```

Comment:
Will this one be deprecated soon?

Reply:
No, those a the exchangers codes that should be implemented